flocker/apiclient/_client.py

```diff
@@ -536,3 +556,17 @@ def to_nodes(result):
         request.addCallback(to_nodes)
 
         return request
+
+    def this_node_uuid(self):
+        era = check_output(["flocker-node-era"])
```

Comment:
Wrote my comment before I saw yours. I still think that it should use Twisted APIs here, since that's the least surprising thing.